bgpug/templates/base.html

```diff
@@ -15,7 +15,9 @@
         <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400italic,400">
         <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Droid+Sans">
         <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lobster">
+        <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <link rel="stylesheet" href="{% static 'bootstrap/dist/css/bootstrap.min.css' %}">
+        <script src="{% static 'bootstrap/dist/js/bootstrap.min.js' %}"></script>
         <link rel="stylesheet" href="{% static 'font-awesome/css/font-awesome.min.css' %}">
```

Comment:
I want to pass all css and js files through pipeline and squash them into single files. So I'll add up to your commits and then merge it. Thanks for fixing the version numbers!